packages/p2p/src/stores/general-store.js

```diff
@@ -30,6 +30,9 @@ export default class GeneralStore extends BaseStore {
     @observable should_show_popup = false;
     @observable user_blocked_until = null;
     @observable is_high_risk_fully_authed_without_fa = false;
+    @observable banner_config = {
+        reduced_order_time: true,
```

Comment:
Why has this been assigned to true?

Reply:
This banner needs to be removed in the next release. For a better handling, I am using a configuration, where I can control the display of banner. The config is not available from API nor we have a firebase where this can be configured. Hence, I am using it in store